### PR TITLE
Add City Lights Theme

### PR DESCRIPTION
**Pull request by nathandcarter21**
### Description

I added a file called city_lights.css to the themes folder with information on the theme.
I edited the list of themes file to also contain my theme.
Links to screenshots: 
https://drive.google.com/file/d/1mQHFt18JGPnczWogHLLQt3iT8Xw2Dos_/view?usp=sharing
https://drive.google.com/file/d/1QHt5yL31M7A5BnWdZxQzhObIgXa3ucpl/view?usp=sharing

Closes #

### CODE REVIEW

File: static/themes/_list.json [hunk 1]
```diff
@@ -674,4 +674,9 @@
     "bgColor": "#464d66",
     "mainColor": "#e0513e"
    }
+   ,{
```
Comment (rizwanmustafa) [inline static/themes/_list.json:677]: It is better to have this comma in the end of the upper line.

**Review by rizwanmustafa — DISMISSED** [review]
As far as the code goes, it is fine.

**Comment by Miodec** [comment]
I would say the colors are similar to moonlight. Plus the text color is not monochrome.

**Comment by Nugget1308** [comment]
It looks good, but can you use a little less brighter pink(or red).

**Pull request closed without merging**